src/components/AmplitudeAnalytics/constants.ts

```diff
@@ -26,15 +27,33 @@ export enum EventName {
 }
 
 export enum CUSTOM_USER_PROPERTIES {
-  WALLET_ADDRESS = 'wallet_address',
-  WALLET_TYPE = 'wallet_type',
-  USER_LAST_SEEN_DATE = 'user_last_seen_date',
-  USER_FIRST_SEEN_DATE = 'user_first_seen_date',
-  WALLET_CHAIN_IDS = 'all_wallet_chain_ids',
   ALL_WALLET_ADDRESSES_CONNECTED = 'all_wallet_addresses_connected',
-  SCREEN_RESOLUTION = 'screen_resolution',
+  ALL_WALLET_CHAIN_IDS = 'all_wallet_chain_ids',
   BROWSER = 'browser',
-  LIGHT_MODE = 'light_mode',
+  DARK_MODE = 'is_dark_mode',
+  SCREEN_RESOLUTION_HEIGHT = 'screen_resolution_height',
+  SCREEN_RESOLUTION_WIDTH = 'screen_resolution_width',
+  WALLET_ADDRESS = 'wallet_address',
+  WALLET_NATIVE_CURRENCY_BALANCE_USD = 'wallet_native_currency_balance_usd',
+  WALLET_CHAIN_IDS_PREFIX = 'wallet_chain_ids_',
```

Comment:
i think prefix/suffix should live separately?

Reply:
sgtm done